test/kb_quast_server_test.py

```diff
@@ -1,30 +1,27 @@
 # -*- coding: utf-8 -*-
-import unittest
+import hashlib
+import inspect
 import os  # noqa: F401
+import shutil
 import time
+import unittest
+import uuid
+from configparser import ConfigParser  # py3 @UnresolvedImport @Reimport
```

Comment:
Don't think the annotations are needed here since there's no try

Reply:
Fixed